renderer/components/editor/buttons/fullscreen.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {FullscreenIcon, ExitFullscreenIcon} from '../../../vectors';
+
+const Fullscreen = ({isFullscreen = false, toggleFullscreen}) => <span onClick={toggleFullscreen} style={{marginLeft: '16px'}}>{isFullscreen ? <ExitFullscreenIcon shadow fill="#FFF" hoverFill="#FFF"/> : <FullscreenIcon shadow fill="#FFF" hoverFill="#FFF"/>}</span>;
```

Comment:
Should we use style jsx here as well, just to be consistent?

Reply:
Sure, i just saw the `fill` and `hoverFill` attributes on svg, maybe remove those then?

Reply:
Sure. I think those are just props for the `svg` component tho. I don't think they are part of a `style` prop

Reply:
Oh, I didn't see that i used the inline styles here, yes. Should i keep the `fill` and `hoverFill` as props or move them so css?

Reply:
Wait, the `fill` and `hoverFill` are already styled jsx https://github.com/wulkano/kap/pull/450/files#diff-0c6cc5100f1fe8cb6cc59c1fb03d3482R48